tests/v2/validation/nodescaling/replace.go

```diff
@@ -93,17 +98,19 @@ func ReplaceRKE1Nodes(t *testing.T, client *rancher.Client, clusterName string,
 	numOfNodesBeforeDeletion, nodeToDelete := MatchNodeToRole(t, client, clusterID, isEtcd, isControlPlane, isWorker)
 
 	for i := range nodeToDelete {
-		logrus.Info("Replacing node: " + nodeToDelete[i].NodeName)
+		logrus.Info("Deleting node: " + nodeToDelete[i].NodeName)
 		err = client.Management.Node.Delete(&nodeToDelete[i])
 		require.NoError(t, err)
 
-		err = clusters.WaitClusterToBeUpgraded(client, clusterID)
+		err = nodestat.IsNodeDeleted(client, nodeToDelete[i].NodeName, clusterID)
 		require.NoError(t, err)
 
-		logrus.Infof("Checking if node %s is replaced", nodeToDelete[i].NodeName)
+		logrus.Infof("Replacing node: " + nodeToDelete[i].NodeName)
+
 		err = nodestat.AllManagementNodeReady(client, clusterID, defaults.ThirtyMinuteTimeout)
 		require.NoError(t, err)
 
+		logrus.Infof("Checking if node %s is replaced", nodeToDelete[i].NodeName)
 		_, err = nodestat.IsNodeReplaced(client, nodeToDelete[i].ID, clusterID, numOfNodesBeforeDeletion)
```

Comment:
same nit here for behavior of isNodeReplaced

Reply:
It's being used in the auto replace test function below.

Reply:
Resolved offline